src/translate.py

```diff
@@ -4161,6 +4177,8 @@ def process_instr(instr: Instruction) -> None:
                     data.meta.inherited or data.meta.function_return
```

Comment:
extend comment above to mention in_pattern in addition to function_return (so note that there'll be trouble if we rely on this mechanism for mips)

Reply:
Adding a comment - but I'm deliberately using the `in_pattern` heuristic for MIPS as well (unless I shouldn't?)

Reply:
It's good for MIPS too, it's just not as effective, because for MIPS we accept arguments set in other basic blocks and for those the meta bit won't have propagated.